### ISSUE TITLE
(powerline-central-evil-theme) stopped working after update

### ISSUE THREAD

**Issue by rubin55** [milkypostman/powerline#165]
Been using powerline with powerline-central-evil-theme like so for the last 2ish years:

;; Enable powerline status bar.
(require 'powerline)
(powerline-center-evil-theme

As of today or yesterday, that stopped working: I get a blank powerline. Commenting out (powerline-center-evil-theme) makes powerline display (albeit not very prettily). Setting another theme, like (powerline-center-theme) seems to work, looking as expected.

To summarize:

With (powerline-center-evil-theme):
![with-powerline-center-evil-theme](https://user-images.githubusercontent.com/6214491/53347699-f17d5800-3919-11e9-830e-fbb4ce3e1f04.png)

With (powerline-center-theme):
![with-powerline-center-theme](https://user-images.githubusercontent.com/6214491/53347794-2093c980-391a-11e9-8c66-dda32fdfe39a.png)

**Comment by milkypostman** [comment]
I just committed a change that looks like it broke you. Will roll that back

**Comment by milkypostman** [comment]
I have reverted what I think was the offending change. Please allow a day for this to propagate to MELPA then let me know if you have more issues or any other ideas about what went wrong. 

Also please post more detailed reproductions instructions.

**Comment by jrosdahl** [comment]
> I have reverted what I think was the offending change

The culprit is #164, not #163. My bad, sorry. (I did not fully test powerline-center-evil-theme since all I got was a blank mode-line before my change, which I now realize was due to not having an evil-mode-line-tag variable set...)

The easy fix is to revert the parts of #164 that apply to powerline-center-evil-theme, but then powerline-center-evil-theme's mode-line will still look a bit peculiar when inactive. I'll see if I can make a better fix.

**Comment by rubin55** [comment]
I can confirm that at least as of today, powerline-central-evil-theme works as expected :-)
